tests/mypy_test.py

```diff
@@ -22,7 +22,7 @@
 parser.add_argument('-v', '--verbose', action='count', default=0, help="More output")
 parser.add_argument('-n', '--dry-run', action='store_true', help="Don't actually run mypy")
 parser.add_argument('-x', '--exclude', type=str, nargs='*', help="Exclude pattern")
-parser.add_argument('-p', '--python-version', type=str, nargs='*',
+parser.add_argument('-p', '--python-version', type=str, action='append',
```

Comment:
Just curious -- why did this need to change? I'm unsure about the difference, or why -x and filter don't need to change.